### PR TITLE
About portal, Support, Rules 

### LINKED ISSUE #1292: [About Portal, Support Information, Laws and Regulations] - displaying actual info on the website
Link information from back-end, that can be edited and configured in the admin-cabinet to the appropriate portal pages.
Style everything as alternation of section and description
use appropriate API's
image
image
Where is no info yet, display such full-width banner
image
Header picture is static

**NOTE!!** DO not create information for Support and laws and regulations, test only on about portal

https://api.oos.dmytrominochkin.cloud/swagger/index.html

Swagger:
1) AboutPortal
2)SupportInformation
3) LawsAndRegulations

### PR DESCRIPTION
**Pull request by sivxkxn**
#1292

### CODE REVIEW

**Review by litvinets — COMMENTED** [review]
use state please

**Review by litvinets — CHANGES_REQUESTED** [review]
can you please fix a bit of styling? It should be the same as before 
image
image

**Review by litvinets — CHANGES_REQUESTED** [review]
Please, can you additionally add routing back on successful update 
Currently the routing is broken 
image

Err: polyfills.07a5c2670fe49c10aa0d.js:1 Unhandled Promise rejection: Cannot match any routes. URL Segment: 'admin-tools/platform/AboutPortal' ; Zone: <root> ; Task: Promise.then ; Value: Error: Cannot match any routes. URL Segment: 'admin-tools/platform/AboutPortal'

**Comment by sivxkxn** [comment]
@litvinets 
1. replaced services with store;
2. added routing after Save changes;
3. left the styles as they were, the font of the text depends on the scale in the browser;
4. fixed build ;

File: src/app/shell/admin-tools/platform/platform-info/info-edit/info-edit.component.html [hunk 1]
```diff
@@ -29,8 +29,8 @@ <h3 class="wrapper-title">РЕДАГУВАННЯ {{editTitle}}</h3>
 
     <div fxLayout="row" fxLayoutAlign="center center" class="footer">
       <button mat-raised-button class="btn btn-cancel" (click)="onBack()">Скасувати</button>
-      <button class="btn" mat-button type="submit"[disabled]="PlatformInfoItemArray.invalid || titleFormControl.invalid" (click)="onSubmit()">Зберегти</button>
+      <a [routerLink]="setRoute()"><button class="btn" mat-button type="submit"[disabled]="PlatformInfoItemArray.invalid || titleFormControl.invalid"  (click)="onSubmit()">Зберегти</button></a>
```
[the remark this answers is not in the document]
Comment (litvinets) [reply src/app/shell/admin-tools/platform/platform-info/info-edit/info-edit.component.html:32]: this.router.navigate([`/admin-tools/platform/`, { queryParams: {page: payload}}]); 
image

**Review by litvinets — CHANGES_REQUESTED** [review]

**Review by litvinets — APPROVED** [review]

File: src/app/shell/admin-tools/platform/platform-info/info-edit/info-edit.component.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (sivxkxn) [reply src/app/shell/admin-tools/platform/platform-info/info-edit/info-edit.component.html]: done, but it is not working as expected

File: src/app/shared/store/admin.state.ts [hunk 2]
```diff
@@ -204,7 +205,7 @@ export class AdminState {
       new MarkFormDirty(false),
       new ShowMessageBar({ message: 'Інформація про портал успішно відредагована', type: 'success' })
     ]);
-    this.router.navigate([`/admin-tools/platform/${payload}`]);
+    this.router.navigate([`/admin-tools/platform/`, { queryParams: {page: payload}}]);
```
Comment (litvinets) [inline src/app/shared/store/admin.state.ts:208]: instaed of payload pass type

**Review by litvinets — CHANGES_REQUESTED** [review]

**Review by litvinets — APPROVED** [review]

**Pull request merged**